pkg/operator/ceph/object/controller.go

```diff
@@ -333,6 +333,7 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 
 	// Return and do not requeue
 	logger.Debug("done reconciling")
+	// TODO: create Ceph Object Store User name cosi-<objectstorename> with a secret containing its credentials for each object store, if not already created
```

Comment:
Still to do in this PR?

Reply:
Will address as seperate PR, current user need to create the user manually

Reply:
Ah, I see. Let's remove this from the code and make a GH issue for it.